client/js/views/GameView.vue

```diff
@@ -1273,15 +1272,15 @@ export default {
       height: 85%;
       font-size: 0.75em;
       letter-spacing: 0.25px;
-      font-family: 'Libre Baskerville', serif;
+      font-family: 'Libre Baskerville', Century Gothic, CenturyGothic, AppleGothic, sans-serif;
```

Comment:
@seriouslysean added font fall backs.

Reply:
Don't we need to import these other fonts? Also what's the difference between Century Gothic and CenturyGothic? If we need the version with spaces we should put it in quotes like `'Century Gothic'`

Reply:
@itsalaidbacklife these should all be Web Safe system fonts I got from here https://www.cssfontstack.com/Century-Gothic.

Since these are system fonts I do not believe we need to import them if we do please let me know and I will look into importing them. 

I can update the font to 'Century Gothic'.